pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1150,4 +1150,7 @@ self: super: {
     };
   };
 
+  # https://github.com/kcsongor/generic-lens/pull/60
+  generic-lens = dontCheck super.generic-lens;
+
```

Comment:
Why not fix the test-suite using the patch in the PR, as in:

```nix
{
  # https://github.com/kcsongor/generic-lens/pull/60
  generic-lens = appendPatch super.generic-lens (pkgs.fetchpatch {
    url = https://github.com/kcsongor/generic-lens/commit/d9af1ec22785d6c21e928beb88fc3885c6f05bed.patch;
    sha256 = "0ljwcha9l52gs5bghxq3gbzxfqmfz3hxxcg9arjsjw8f7kw946xq";
  });
}
```

Reply:
Honestly, because I didn't know you could do that. Is it OK to force-push to do that instead?

Reply:
Yes no problem.

Reply:
Done